shared/constants/types/teams.tsx

```diff
@@ -22,7 +22,7 @@ export type PublicitySettings = {
   publicityTeam: boolean
 }
 
-export type ActivityLevel = 'active' | 'recently' | 'extinct'
+export type ActivityLevel = 'active' | 'recently'
```

Comment:
not all channels have an activity level, by this definition. Can we have a 'none' or something to differentiate "not showing because no data" and "not showing because not active"?

Reply:
Good point, I'll add that